src/diffusers/loaders/lora_pipeline.py

```diff
@@ -3254,6 +3254,313 @@ def unfuse_lora(self, components: List[str] = ["transformer", "text_encoder"], *
         super().unfuse_lora(components=components)
 
 
+class LTXVideoLoraLoaderMixin(LoraBaseMixin):
```

Comment:
I think this could be a direct copy-paste of the `MochiLoraLoader` no? I am unable think of any differences between the two.

Reply:
This is currently the copy-paste of CogVideoXLoraLoaderMixin with no changes. I think they're both the same, but will take a look at diffs

Reply:
Oh that works too.